src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -335,4 +341,23 @@ private function configureStorage(array $config, ContainerBuilder $container, Lo
 
         $container->setAlias('sulu_media.storage', 'sulu_media.storage.' . $storage)->setPublic(true);
     }
+
+    private function getSupportedMimeTypes($ghostScriptPath, $ffmpegBinary, $ffprobeBinary)
+    {
+        $mimeTypes = ['image/*'];
+
+        if ($ffmpegBinary
+            && $ffprobeBinary
+            && shell_exec('which ' . $ffmpegBinary)
```

Comment:
`which` will not work on windows i guess

Reply:
I looked it up, seems to exist on windows as well. The thing is that I don't want the actual command to be executed.